versioned_docs/version-22.10/monitoring/anomaly-detection.md

```diff
@@ -5,56 +5,61 @@ title: Anomaly detection
+When the behavior of the service deviates from the expected model, these thresholds are passed (i.e. the metric goes below the lower threshold or goes over the upper threshold). The service goes into a CRITICAL state and a notification is sent. Exemple: a server usually has little traffic at night. One night, Centreon detects network flows higher than normal and triggers an alert. This makes the company aware that the server has been hacked.
+
+## How it works
+
+1. Collected data is sent to the Centreon SaaS platform.
 
-Collected data is sent to the Centreon Cloud platform in order to be able to
-compute a regular behavior model thanks to the history of this data.
+2. Centreon computes a regular behavior model thanks to the history of this data.
 
-Once the model has been calculated, predictions are then generated and retrieved
-on the on-premise Centreon platform.
+3. Once the model has been calculated, predictions are then generated and retrieved
+by your Centreon platform.
 
-These predictions will serve as floating thresholds which will then be used by
+4. The predictions act as floating thresholds which will then be used by
 the monitoring engine to compare the collected value with the predicted
-thresholds to highlight deviations and generate alerts.
+thresholds to highlight deviations and generate alerts. Data received during downtimes are not taken into account when computing predictions so as not to distort the analysis.
 
-![image](../assets/monitoring/anomaly/centreon_cloud.png)
+5. Models are recomputed regularly and improve over time.
 
 ## Prerequisites
 
-The Centreon Anomaly Detection module requires the following prerequisites:
+To use Anomaly Detection, you will need:
 
-  - Centreon in minimum version 20.04
-  - A token provided by Centreon to access the Centreon Cloud platform
-  - An Internet connection from the Centreon Central server
-  - The SHELL environment variable [LC_ALL](https://www.gnu.org/software/gettext/manual/html_node/Locale-Environment-Variables) must not be set, or be set to `C`. To check the value of this variable, enter:
+- A token provided by Centreon to access the Centreon SaaS platform,
+- An internet connection from the Centreon central server
+- The SHELL environment variable [LC_ALL](https://www.gnu.org/software/gettext/manual/html_node/Locale-Environment-Variables) must not be set, or be set to `C`. To check the value of this variable, enter:
 
-    ```
-    echo $LC_ALL
-    ```
-      
-  - Prediction best works with monitored services that present a seasonal
-    behaviour as shown below:
-
-![image](../assets/monitoring/anomaly/simple_scheme.png)
+   ```
+   echo $LC_ALL
+   ```
 
 ## Installation
 
-### Installing packages
+### Step 1: Installing packages
 
-Add additional repository:
+1. Add an additional repository:
 
 <Tabs groupId="sync">
 <TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">
 
-Nothing to do
```

Comment:
The "Nothing to do" and the command for Centos 7 are shifted to the left from the content of point 2

Reply:
This is because of the set of tabs. That's how Docusaurus works. It's very ugly but I can't do anything about it, I'm afraid.